xla/service/gpu/matmul_utils.h

```diff
@@ -43,6 +43,31 @@ limitations under the License.
 namespace xla {
 namespace gpu {
 
+
+inline void GetXlaPrecisionConfigNumericFlags(const mlir::ArrayAttr& precision_config,
```

Comment:
How does this function relate to the one with the same name in xla/util.h?

Reply:
It is the same function, but with a different data type (at some point precision config gets converted into mlir::ArrayAttr).